templates/home.html

```diff
@@ -39,12 +39,14 @@
     <div>
         <h3 class="mb-4">Welcome<span v-cloak v-if="username"> back</span> to Wikipedia Speedruns<span  v-cloak v-if="username">, [[username]]</span>!</h3>
 
-        <div class="alert alert-warning" v-cloak role="alert" v-html="streakText" v-if="loggedIn && streakText"></div>
-
-        <div class="alert alert-info" v-cloak role="alert" v-if="loggedIn">
-            We've just added <b>achievements</b> to the game! Visit your profile page <a v-bind:href="'/profile/' + username">here</a> to check your current progress.
+        <!---->
+        <div class="alert alert-info" v-cloak role="alert" v-if="new Date() < new Date('2023-01-31')">
```

Comment:
Don't think we need to add an alert for this, draws unnecessary attention imo. Can add it to the game rules instead. Although, calling out that our game isn't affected by edits discourages mal edits which is what we want so maybe it's fine.